### DISCUSSION TITLE
Unable to load only related records in belongsTo field using dependsOn

### DISCUSSION THREAD

**Discussion by FSElias** [laravel/nova-issues#4453] [Ideas & Feature Requests]
- Laravel Version: 9.17
- Nova Version: 4.9
- PHP Version: 8.1.2

### Description:
Unable to load only related records in belongsTo field using dependsOn

### Detailed steps to reproduce the issue on a fresh Nova installation:
This is a simple example of my code, I am not sure if I am doing something wrong?
And don't want to change BelongsTo::make('License') to a Select field.

```php
//Nova/Order.php
    public function fields(NovaRequest $request)
    {
        return [
            BelongsTo::make('User'),
            BelongsTo::make('License')
                     ->dependsOn(
                         ['user'],
                         function (BelongsTo $field, NovaRequest $request, FormData $formData) {
                             $field->belongsToId = $formData->user; //<<<< this line
                         }
                     ),
        ];
    }
```

```php
//Models/Order.php
public function user()
{
    return $this->belongsTo(User::class);
}

public function license()
{
    return $this->belongsTo(License::class);
}
```

```php
//Models/User.php
public function licenses()
{
    return $this->hasMany(License::class);
}
```

```php
//Models/License.php
public function user()
{
    return $this->belongsTo(User::class);
}
```

**Comment by crynobone** [comment]
`belongsToId` is only used for initializing form for Update. You can set `default()` but that doesn't limit users from using other BelongsTo results.

**Comment by FSElias** [comment]
Ok, is there a method/workaround/planning for multi belongsTo fields dependencies? I had two projects that needed that feature, one of them required "showCreateRelationButton" yet I had to use Select fields
